GitCommands/Settings/AppSettings.cs

```diff
@@ -1499,6 +1499,12 @@ public static bool FillRefLabels
             set => SetBool("FillRefLabels", value);
         }
 
+        public static bool ReduceGraphCurves
+        {
+            get => GetBool("ReduceGraphCurves", true);
+            set => SetBool("ReduceGraphCurves", value);
+        }
```

Comment:
I'm thinking more and more that we need to have an "Experimental" section in settings where we can expose settings and easily toggle those.